src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationModalMap.js

```diff
@@ -238,14 +245,26 @@ const ImageAnnotationModalMap = ({
   }, [highlightedPoints, selectedPoints, hasMapLoaded])
 
   return (
-    <div
-      style={{
-        width: dataToReview.original_image_width * imageScale,
-        height: dataToReview.original_image_height * imageScale,
-        marginTop: '2rem',
-      }}
-      ref={mapContainer}
-    />
+    <>
+      <div
```

Comment:
could add modal style as a const vs inline

Reply:
hard to see properly in the editor but it's this bit:

<img width="538" alt="Screenshot 2024-08-26 at 3 22 34 PM" src="https://github.com/user-attachments/assets/3929791c-f3ae-46fa-b02e-16ec88e831f1">